### ISSUE TITLE
Link between layers (not just artboards)

### ISSUE THREAD

**Issue by sgraden** [abynim/UserFlows.sketchplugin#99]
I think the functionality of a tool where you select two items and draw a permanent connection between them in sketch can be really useful in a huge variety of situations. 

In my particular case, I am just doing a text/box based user flow diagram. Normally I would have to draw all of the boxes and then draw a line between each one. If something changes I need to move everything around and redraw the lines manually.

With UserFlows plugin (enabled with layer/group to layer/group links) it would be much simpler to mover everything around, break one link, add new content, and relink.

P.S. Thank you for all the hard work.

**Comment by alessandrodallafina** [comment]
Totally agreed. It'd be awesome linking layers, groups or objects between them without using any artboard.

**Comment by CVSccorpus** [comment]
was checking to see if this was requested. glad to see it was. the instance where i would use this would be to show anchored links.

**Comment by rolandcedo** [comment]
Agreed. I think this would be able to tell a more specific story when it comes to microinteractions.
